site/man/rabbitmq-streams.8.html

```diff
@@ -303,7 +314,155 @@ <h3 class="Ss" title="Ss" id="Stream_plugin"><a class="permalink" href="#Stream_
+    [<div class="Op"><code class="Fl" title="Fl">--initial-cluster-size</code>
+    <var class="Ar" title="Ar">initial-cluster-size</var></div>]</dt>
+  <dd>
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">super-stream</var></dt>
+      <dd>The name of the super stream to create.</dd>
+      <dt><var class="Ar" title="Ar">vhost</var></dt>
+      <dd>The name of the virtual host to create the super stream into.</dd>
+      <dt><var class="Ar" title="Ar">partitions</var></dt>
+      <dd>The number of partitions the super stream will have.</dd>
+      <dt><var class="Ar" title="Ar">routing-keys</var></dt>
+      <dd>Comma-separated list of routing keys.</dd>
+      <dt><var class="Ar" title="Ar">max-length-bytes</var></dt>
+      <dd>The maximum size of partition streams, example values: 20gb,
+        500mb.</dd>
+      <dt><var class="Ar" title="Ar">max-age</var></dt>
+      <dd>The maximum age of partition stream segments, using the ISO 8601
+          duration format, e.g. PT10M30S for 10 minutes 30 seconds, P5DT8H for 5
+          days 8 hours.</dd>
+      <dt><var class="Ar" title="Ar">stream-max-segment-size-bytes</var></dt>
+      <dd>The maximum size of partition stream segments, example values: 500mb,
+          1gb.</dd>
+      <dt><var class="Ar" title="Ar">leader-locator</var></dt>
+      <dd>Leader locator strategy for partition streams. Possible values are:
+        <ul class="Bl-bullet Bl-compact">
+          <li>client-local</li>
+          <li>balanced</li>
+        </ul>
+        The default is <code class="Cm" title="Cm">balanced</code></dd>
+      <dt><var class="Ar" title="Ar">initial-cluster-size</var></dt>
+      <dd>The initial cluster size of partition streams.</dd>
+    </dl>
+    <div class="Pp"></div>
+    Create a super stream.</dd>
+  <dt><a class="permalink" href="#delete_super_stream"><code class="Cm" title="Cm" id="delete_super_stream">delete_super_stream</code></a>
+    <var class="Ar" title="Ar">super-stream</var>
+    [<div class="Op"><code class="Fl" title="Fl">--vhost</code>
+    <var class="Ar" title="Ar">vhost</var></div>]</dt>
+  <dd>
+    <dl class="Bl-tag">
+      <dt><var class="Ar" title="Ar">super-stream</var></dt>
+      <dd>The name of the super stream to delete.</dd>
+      <dt><var class="Ar" title="Ar">vhost</var></dt>
+      <dd>The virtual host of the super stream.
+        <div class="Pp"></div>
+        Delete a super stream.</dd>
+    </dl>
+    <div class="Pp"></div>
+  </dd>
+  <dt><a class="permalink" href="#list_stream_consumer_groups"><code class="Cm" title="Cm" id="list_stream_consumer_groups">list_stream_consumer_groups</code></a>
+    [<div class="Op"><code class="Fl" title="Fl">-p</code>
+    <var class="Ar" title="Ar">vhost</var></div>]
+    [<div class="Op"><var class="Ar" title="Ar">groupinfoitem
+    ...</var></div>]</dt>
+  <dd>
+    <div class="Pp"></div>
+    Lists groups of stream single active consumers for a vhost.
```

Comment:
What does " Lists groups of stream single active consumers for a vhost." mean? 

Is it "Lists the groups of single active consumers in a stream for a vhost" if so please change to this - thanks